.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: edvinas108/meme-grid:latest
+    steps:
+      - checkout
+      - run:
+          -name: Run tests
+          -command: ./gradlew test
+workflows:
+  version: 2
+  test:
+    jobs:
+      - test
```

Comment:
I'd suggest to use the `check` task, if in the future we add static analysis and such, it will run as well and not only the tests.

Reply:
Changed to use gradle check